src/main/java/formflow/library/FormFlowController.java

```diff
@@ -41,10 +45,23 @@ public abstract class FormFlowController {
     this.messageSource = messageSource;
   }
 
+  /**
+   * Saves a {@link Submission}
+   *
+   * @param submission submission saved through the submission repository service.
+   * @return A decrypted submission
```

Comment:
I don't think you needed to say "decrypted" here. 

Instead, maybe: "The saved submission object"

Reply:
done